library/lib.ml

```diff
@@ -440,6 +440,21 @@ let add_section_context ctx =
        check_same_poly true vars;
        sectab := (Context ctx :: vars,repl,abs)::sl
 
+let is_polymorphic_univ u =
+  let exception Found of bool in
+  try
+    let open Univ in
+    List.iter (fun (vars,_,_) ->
+        List.iter (function
+            | Variable (_,_,poly,(univs,_)) ->
+              if LSet.mem u univs then raise (Found poly)
+            | Context (univs,_) ->
+              if LSet.mem u univs then raise (Found true)
+          ) vars
+      ) !sectab;
+    false
+  with Found b -> b
```

Comment:
I don't know if this is the nicest way to figure out if a universe is a section polymorphic universe but it should work well enough.